doc/source/whatsnew/v1.2.0.rst

```diff
@@ -57,6 +57,13 @@ Other enhancements
 -
 -
 
+.. _whatsnew_120.api_breaking.experimental:
+
+Changes to experimental APIs
+----------------------------
+
+- :meth:`pandas.api.extensions.ExtensionDtype.construct_array_type` has changed from a classmethod to a regular method to support one dtype being used for multiple arrays. To migrate, change your definition to a regular method and ensure that your method is called on instances rather than the class (:issue:`36126`).
```

Comment:
To be clear, as long as you are not calling it yourself on the class object, and only implementing it, it should directly break existing dtypes ? (a class method can also be called from an instance)

(otherwise I think we should do it in a non-breaking way, eg by taking a different name)